### PR TITLE
add job and press page responsive

### PR DESCRIPTION
**Pull request by mailtime-wing**

### CODE REVIEW

File: src/pages/jobs.js [hunk 1]
```diff
@@ -11,28 +11,31 @@ import Job from "@/components/JobSection"
...
 const background = theme => css`
   background-image: ${theme.linearGradients.greenDarkToLight};
   margin-bottom: 26.5rem;
 `
 
-const hiringPadding = css`
-  padding-top: 26.3rem;
-  padding-left: 5.7rem;
+const hiringPadding = isMobile => css`
+  padding-top: ${isMobile ? "8.8rem" : "26.3rem"};
```
Comment (chochihim) [inline src/pages/jobs.js]: use media-query in css

File: src/pages/jobs.js [hunk 2]
```diff
@@ -11,28 +11,31 @@ import Job from "@/components/JobSection"
...
   text-align: left;
 `
 
-const jobsSection = css`
+const jobsSection = isMobile => css`
   margin: 0 auto;
-  width: 102.4rem;
-  padding-left: 6.9rem;
-  padding-right: 1.2rem;
+  width: ${isMobile ? "100%" : "102.4rem"};
```
Comment (chochihim) [inline src/pages/jobs.js]: use media-query in css

File: src/pages/press.js [hunk 3]
```diff
@@ -7,28 +7,31 @@ import SEO from "@/components/seo"
...
 import Press from "@/components/PressSection"
 
-const background = theme => css`
+import useMedia from "@/hooks/useMedia"
+
+const background = (theme, isMobile) => css`
   padding: 0;
   background-image: ${theme.linearGradients.greenDarkToLight};
-  margin-bottom: 30.6rem;
+  margin-bottom: ${isMobile ? "23rem" : "30.6rem"};
```
Comment (chochihim) [inline src/pages/press.js]: use media-query in css

File: src/pages/press.js [hunk 4]
```diff
@@ -7,28 +7,31 @@ import SEO from "@/components/seo"
...
 `
 
-const shoutOutPadding = css`
-  padding-top: 26.3rem;
-  padding-left: 5.7rem;
+const shoutOutPadding = isMobile => css`
```
Comment (chochihim) [inline src/pages/press.js]: use media-query in css

File: src/pages/jobs.js (hunk 1 above)
Comment (mailtime-wing) [reply src/pages/jobs.js]: fixed.

File: src/pages/jobs.js (hunk 2 above)
Comment (mailtime-wing) [reply src/pages/jobs.js]: fixed.

File: src/pages/press.js (hunk 4 above)
Comment (mailtime-wing) [reply src/pages/press.js]: fixed.

File: src/pages/press.js (hunk 3 above)
Comment (mailtime-wing) [reply src/pages/press.js]: fixed.

**Pull request merged**